common/src/configuration/bootstrap.rs

```diff
@@ -113,6 +113,8 @@ pub struct ConfigBootstrap {
     /// This will clean out the orphans db at startup
     #[structopt(long, alias("clean_orphans_db"))]
     pub clean_orphans_db: bool,
+    #[structopt(long, alias("enable_mininf"))]
```

Comment:
There could be a spelling mistake here?
`enable_mining` vs `enable_mininf`

Reply:
Thanks, that definitely was.